src/ui/About.tsx

```diff
@@ -21,6 +27,8 @@ export class About extends React.Component<Props> {
     }
 
     private renderView() {
+        const idealAvatarResolution = Math.round(devicePixelRatio * 42);
```

Comment:
Can we export this 42 from the SCSS and import it here, like we do in `variables.scss`?

Reply:
Yeah, though I wasn't fully sure how to do this? What do I do with the "px" part?

Reply:
I think it just works with `parseFloat` 
https://github.com/LiveSplit/LiveSplitOne/blob/909d61568d244fab5de77045085b845232a59fbb/src/css/variables.scss#L36
https://github.com/LiveSplit/LiveSplitOne/blob/909d61568d244fab5de77045085b845232a59fbb/src/ui/LiveSplit.tsx#L30

Reply:
Oh, similar code uses `parseFloat`, which just silently gets rid of it.

Reply:
Maybe we can add a comment that explains that behavior

Reply:
Forgot about the comment, but I did fix it.